### PR TITLE
[CBRD-25398] The 'memmon' is not displayed in the help message of the cubrid utility

### PR DESCRIPTION
**Pull request by H2SU**
http://jira.cubrid.org/browse/CBRD-25398

Purpose

When running the cubrid utility, help messages are displayed. These help messages show the list of available utilities. Despite the addition of the 'memmon' utility, it is not included in the list of help messages. However, when running 'cubrid memmon', the help message for the 'memmon' utility is displayed correctly.

Implementation

N/A

Remarks

N/A

### CODE REVIEW

**Comment by joohok** [comment]
as far as I know that there are about 14 locales. Why haven't the other languages been modified?

**Comment by hornetmj** [comment]
@H2SU When performing grep in the 'msg' directory, there are still unmodified 'utils.msg' files. Please modify them together.

```
$ grep -rcH '    memmon' */utils.msg
de_DE.utf8/utils.msg:1
en_US.utf8/utils.msg:3
en_US/utils.msg:3
es_ES.utf8/utils.msg:1
fr_FR.utf8/utils.msg:1
it_IT.utf8/utils.msg:1
ja_JP.utf8/utils.msg:1
km_KH.utf8/utils.msg:3
ko_KR.euckr/utils.msg:1
ko_KR.utf8/utils.msg:1
ro_RO.utf8/utils.msg:1
tr_TR.utf8/utils.msg:1
vi_VN.utf8/utils.msg:3
zh_CN.utf8/utils.msg:1
```

**Comment by H2SU** [comment]
The 'utils.msg' file for the locale, which had not been modified before, has been modified.

```
$grep -rcH '    memmon' */utils.msg
de_DE.utf8/utils.msg:3
en_US.utf8/utils.msg:3
en_US/utils.msg:3
es_ES.utf8/utils.msg:3
fr_FR.utf8/utils.msg:3
it_IT.utf8/utils.msg:3
ja_JP.utf8/utils.msg:3
km_KH.utf8/utils.msg:3
ko_KR.euckr/utils.msg:3
ko_KR.utf8/utils.msg:3
ro_RO.utf8/utils.msg:3
tr_TR.utf8/utils.msg:3
vi_VN.utf8/utils.msg:3
zh_CN.utf8/utils.msg:3
```

File: msg/ko_KR.euckr/utils.msg
Comment (joohok) [inline msg/ko_KR.euckr/utils.msg:1]: The locale is incorrect. Please fix it.

Comment (H2SU) [reply msg/ko_KR.euckr/utils.msg:1]: has been  fixed.

**Review by hornetmj — APPROVED** [review]

**Review by YeunjunLee — APPROVED** [review]

**Review by joohok — APPROVED** [review]

**Pull request merged**